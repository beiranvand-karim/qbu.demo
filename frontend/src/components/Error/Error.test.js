/* global describe, it, jest, expect*/
import {shallow} from "enzyme"
import {Error} from "./"
import React from "react"
import renderer from "react-test-renderer"
import {MemoryRouter} from "react-router-dom"

describe('<Error />', () => {
   it('should render without crashing', () => {
      shallow(<Error/>)
   });
   it('should matches the snapshot', () => {
      const tree = renderer.create(<MemoryRouter><Error/></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});