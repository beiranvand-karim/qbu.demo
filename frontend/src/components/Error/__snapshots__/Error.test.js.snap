// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> should matches the snapshot 1`] = `
<div>
  error: 
</div>
`;
